tests/PKSim.Tests/IntegrationTests/FlatParameterInContainsRepositorySpecs.cs

```diff
@@ -33,7 +34,11 @@ public class When_checking_the_database_consistency_regarding_parameter_flags :
       [Observation]
       public void all_hidden_parameters_should_be_read_only()
       {
-         var parameters = _allParameters.Where(x => !x.Visible).Where(x => !x.ReadOnly).ToList();
+         var parameters = _allParameters.Where(x => !x.Visible)
+            .Where(x => !x.ReadOnly)
+            .Where(x => x.ParameterName != CoreConstants.Parameters.SOLUBILITY_TABLE)
```

Comment:
This is new. This tests was failing because by defaut, solubility table is hidden but not readonly